views/finance/contracted_arr_reporting.view.lkml

```diff
@@ -0,0 +1,604 @@
+    sql: ${expire} ;;
+  }
+
+  dimension: geo {
+    type: string
+    sql: ${TABLE}."GEO" ;;
+  }
+
+  dimension: government {
+    type: yesno
+    sql: ${TABLE}."GOVERNMENT" ;;
+  }
+
+  dimension: industry {
+    type: string
+    sql: ${TABLE}."INDUSTRY" ;;
+  }
+
+  dimension: late_renewal {
+    type: number
+    value_format: "$#,##0;($#,##0)"
+    description: "Renewal closed based on the amount of latest license expired"
+    sql: ${TABLE}."LATE_RENEWAL" ;;
+  }
+
+  measure: total_late_renewal {
+    type: sum
+    value_format: "$#,##0;($#,##0)"
+    drill_fields: [close_day_date,account_name,late_renewal,account_id,account_owner,parent,tier,bin_avg_arr,tcv,geo,country,government,company_type, industry,license_beg_date,license_end_date,term]
+    sql: ${late_renewal} ;;
+  }
+
+  dimension_group: license_beg {
+    type: time
+    timeframes: [
+      raw,
+      date,
+      week,
+      month,
+      quarter,
+      year
+    ]
+    convert_tz: no
+    datatype: date
+    sql: ${TABLE}."LICENSE_BEG" ;;
+  }
+
+  dimension_group: license_end {
+    type: time
+    timeframes: [
+      raw,
+      date,
+      week,
+      month,
+      quarter,
+      year
+    ]
+    convert_tz: no
+    datatype: date
+    sql: ${TABLE}."LICENSE_END" ;;
+  }
+
+  dimension: license_month {
+    type: string
+    sql: ${TABLE}."LICENSE_MONTH" ;;
+  }
+
+  dimension: license_quarter {
+    type: string
+    sql: ${TABLE}."LICENSE_QUARTER" ;;
+  }
+
+  dimension: new {
+    type: number
+    value_format: "$#,##0;($#,##0)"
+    description: "ARR signed up with new accounts"
+    sql: ${TABLE}."NEW" ;;
+  }
+
+  measure: total_new {
+    type: sum
+    value_format: "$#,##0;($#,##0)"
+    drill_fields: [close_day_date,account_name,new,account_id,account_owner,parent,tier,bin_avg_arr,tcv,geo,country,government,company_type, industry,license_beg_date,license_end_date,term]
+    sql: ${new} ;;
+  }
+
+  measure: avg_new {
+    type: average
+    value_format: "$#,##0;($#,##0)"
+    drill_fields: [close_day_date,account_name,new,account_id,account_owner,parent,tier,bin_avg_arr,tcv,geo,country,government,company_type, industry,license_beg_date,license_end_date,term]
+    sql: ${new} ;;
+  }
+
+  dimension: opportunity_description {
+    type: string
+    sql: ${TABLE}."OPPORTUNITY_DESCRIPTION" ;;
+  }
+
+  dimension: parent {
+    type: string
+    description: "Parent company account belongs to"
+    sql: ${TABLE}."PARENT" ;;
+  }
+
+  dimension: parent_id {
+    type: string
+    sql: ${TABLE}."PARENT_ID" ;;
+  }
+
+  dimension: plan {
+    type: string
+    sql: ${TABLE}."PLAN" ;;
+  }
+
+  dimension: previous_expire {
+    type: number
+    value_format: "$#,##0;($#,##0)"
+    description: "ARR that expired most recently"
+    sql: ${TABLE}."PREVIOUS_EXPIRE" ;;
+  }
+
+  measure: total_preivous_expire {
```

Comment:
total_preivous_expire -> total_previous_expire

Reply:
Thanks, Catalin for catching the spelling